iotdb-core/datanode/src/test/java/org/apache/iotdb/db/metric/MetricServiceTest.java

```diff
@@ -55,30 +50,31 @@ public class MetricServiceTest {
       MetricConfigDescriptor.getInstance().getMetricConfig();
   private static AbstractMetricService metricService = new DoNothingMetricService();
 
-  @Test
-  public void testMetricService() {
-    for (MetricFrameType type : MetricFrameType.values()) {
-      // init metric service
-      metricConfig.setMetricFrameType(type);
-      metricConfig.setMetricLevel(MetricLevel.IMPORTANT);
-      metricService = new DoNothingMetricService();
-      metricService.startService();
-
-      // test metric service
-      assertTrue(metricService.getMetricManager().isEnableMetricInGivenLevel(MetricLevel.CORE));
-      assertTrue(
-          metricService.getMetricManager().isEnableMetricInGivenLevel(MetricLevel.IMPORTANT));
-      assertFalse(metricService.getMetricManager().isEnableMetricInGivenLevel(MetricLevel.NORMAL));
-      assertFalse(metricService.getMetricManager().isEnableMetricInGivenLevel(MetricLevel.ALL));
-
-      testNormalSituation();
-
-      testOtherSituation();
-
-      // stop metric module
-      metricService.stopService();
-    }
-  }
+  //  @Test
+  //  public void testMetricService() {
```

Comment:
Do not comment test.
BTW, I think this test(line58 - line 75) is still useful

Reply:
+1, maybe we should leave the test of micrometer implement

Reply:
Has fixed.